src/main/java/fr/insee/genesis/domain/service/rawdata/LunaticJsonRawDataService.java

```diff
@@ -156,7 +157,73 @@ public DataProcessResult processRawData(String campaignName, List<String> interr
                 batchNumber++;
             }
         }
-        return new DataProcessResult(dataCount, formattedDataCount);
+        return new DataProcessResult(dataCount, formattedDataCount, errors);
+    }
+
+    @Override
+    public DataProcessResult processRawData(String questionnaireId) throws GenesisException {
+        int dataCount=0;
+        int formattedDataCount=0;
+        DataProcessingContextModel dataProcessingContext =
+                dataProcessingContextService.getContextByPartitionId(questionnaireId);
```

Comment:
getContextBy ? use partition or collectionInstrumentId <= I think we should use collectionInstrumentId